stl/inc/ranges

```diff
@@ -34,8 +36,6 @@ _STD_END
 _STL_RESTORE_CLANG_WARNINGS
 #pragma warning(pop)
 #pragma pack(pop)
-#else // ^^^ Concepts support / no Concepts vvv
-#pragma message("The contents of <ranges> are only available with C++20 concepts support.")
 #endif // __cpp_lib_concepts

```

Comment:
Should these be `defined(__cpp_lib_concepts)`?

Reply:
Good question. Our current practice is inconsistent (because we've added arrow comments over the years). Your PR currently follows one of our patterns - repeating the macro name as the `#endif` comment, despite the arrow comment on the `#else`. So this would be fine to leave as-is. You could also consistently use a matching arrow, which would be `// ^^^ defined(__cpp_lib_concepts) ^^^` here. (The one thing we don't do is pair `#ifndef MEOW` with `#endif // defined(MEOW)`.)

Reply:
Well I assume they would eventually be replaced with `_HAS_CXX20` at some point. So I guess It would eventually solve itself?